docs/info.qmd

```diff
@@ -13,18 +13,21 @@ The GUI can be extended by users to implement their own data workflows.
 - `tabBrowser`: (Default) a main table to browse through collections, metadata and ACL manipulation, data deletion and single file up and downloads,
 - `tabUpDownload`: a two-pane upload/download tab
 - `tabELNData`: Uploading data and linking it to an Electronical Labnotebook experiment
-- `tabDataBundle`: (un)bundle datasets from/to four supported formats (depending on extra iRODS microservices)
+- `tabAmberWorkflow`: Up ad downloading data to and from Amberscript from iRODS
+- `tabDataBundle`: (Experimental)(un)bundle datasets from/to four supported formats (depending on extra iRODS microservices)
```

Comment:
I would not consider this experimental.  Currently, it supports only TAR files with built-in microservices, no compression, no custom microservices.  iRODS will eventually add microservice support for compression and other formats.  When is unknown.